pyproject.toml

```diff
@@ -124,10 +126,19 @@ test_core = [
 
     # for github test : probeinterface and neo from master
     # for release we need pypi, so this need to be commented
-    # "probeinterface @ git+https://github.com/SpikeInterface/probeinterface.git",
-    # "neo @ git+https://github.com/NeuralEnsemble/python-neo.git",
+    "probeinterface @ git+https://github.com/SpikeInterface/probeinterface.git",
+    "neo @ git+https://github.com/NeuralEnsemble/python-neo.git",
+]
+
+test_extractors = [
+    # Functions to download data in neo test suite
+    "pooch>=1.8.2",
+    "datalad>=1.0.2",
+    "probeinterface @ git+https://github.com/SpikeInterface/probeinterface.git",
+    "neo @ git+https://github.com/NeuralEnsemble/python-neo.git",
 ]
 
+
```

Comment:
these cen also be factored out at module level:

- `test_sortingcomponents`
```
    "xarray",
    "huggingface_hub",
```

- `test_sorters`
```
    # tridesclous
    "numba",
    "hdbscan>=0.8.33",
``` 
- `test_widgets`
```
    # for sortingview backend
    "sortingview",
```


What do you guys think?

Reply:
Maybe in a different PR

Reply:
I think this is a great idea, but is a different PR from this current one :)  Since we have the dev install this will lighten that install and then the developer can layer on specific module dependencies if they want to work on it or just have the core developer dependencies if they don't.

Reply:
Yes, @alejoe91, moving slowly in that direction. Will link this comment in the roadmap for testing improvement.

Reply:
https://github.com/SpikeInterface/spikeinterface/issues/3100

Reply:
@alejoe91 , what aobut postprocessing and comparisons?